requirements.txt

```diff
@@ -1,5 +1,5 @@
-compas
-pyside2
-pyopengl
-pyopengl-accelerate
-freetype-py
\ No newline at end of file
+# compas
+# pyside2
+# pyopengl
+# pyopengl-accelerate
+# freetype-py
```

Comment:
Should we just delete these requirements if they are not necessary any more?

Reply:
yes indeed

Reply:
hmm seems it is necessary for the installation process for the tests